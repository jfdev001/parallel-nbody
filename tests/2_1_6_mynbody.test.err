Reservation number 3913325: Reserved 2 hosts for 900 seconds 
Run on 2 hosts for 960 seconds from Tue Jan 17 17:49:11 2023
: node063/0 node064/0 
Running N-body with 6 bodies and 1000 steps

N-body took: 0.005 seconds
Performance N-body: 0.09 GFLOPS
[node063:13664] 1 more process has sent help message help-mpi-errors.txt / mpi_errors_are_fatal
[node063:13664] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
