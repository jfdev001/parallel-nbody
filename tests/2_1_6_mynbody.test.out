   313.023      9.014      5.214      6.626      4.577      2.839
   875.978    766.024   -503.221   -285.635     -4.339      4.824
   512.017    214.989  -3856.220  13162.772      3.368     -2.135
   234.008     45.986   1341.955   1023.254      1.620     -2.843
   842.002    746.979   -824.728  -1648.085      0.427     -4.246
   502.013    248.995   3857.335 -12252.732      2.646     -1.027
[node063:13673] *** An error occurred in MPI_Request_free
[node063:13673] *** reported by process [3706650625,0]
[node063:13673] *** on communicator MPI_COMM_WORLD
[node063:13673] *** MPI_ERR_REQUEST: invalid request
[node063:13673] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[node063:13673] ***    and potentially your MPI job)
STATUS=PASSED
